### PR TITLE
Added custom error pages

### PR DESCRIPTION
**Pull request by arun-kushwaha04**
## Fixes Issue
Feature added as mentioned in issue #75 

## Changes proposed
- Added error page to handle 404 error code
- Error page added to handle internal server error

## Check List (Check all the applicable boxes) 
- [X] My code follows the code style of this project.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Page displayed when some internal error occurs

Screenshot from 2023-02-10 05-20-05
## Page displayed when a 404 error occurs (dark mode)
Screenshot from 2023-02-10 05-21-49
## Page displayed when a 404 error occurs (light mode)
Screenshot from 2023-02-10 05-21-54

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Cahllagerfeld) [inline .hintrc:1]: Can you please remove that file from the PR 🙏

**Review by Cahllagerfeld — COMMENTED** [review]
Additional: Do we need a 500 - gif for darkmode?

All in all it's a pretty nice implementation!

Just out of curiosity, where did you get the gifs from? :)

**Comment by arun-kushwaha04** [comment]
> Additional: Do we need a 500 - gif for darkmode?
> 
> All in all it's a pretty nice implementation!
> 
> Just out of curiosity, where did you get the gifs from? :)

I guess when a 500 (Internal error occurs) there no state information for dark or light mode although i am not confident on this.  It only render the default error that i replaced with current one. 

I found these gifs on `tenor.com`, had to edit them for dark and light mode.

**Review by Cahllagerfeld — APPROVED** [review]
Nice work!

**Pull request merged**